### ISSUE TITLE
Cost for completeness

### ISSUE THREAD

**Issue by azhe825** [ai-se/ML-assisted-SLR#65]
Kitchenham data:
content/abs/total: 45/132/1704
Review cost: CR for content review, AR for title and abstract review

 - review until 95% abs relevant retrieved (abs relevant as label): 
    + cost: 126 CR + 1370 AR
    + result: 

 - review until 95% content relevant retrieved (abs relevant as label): 
    + cost: 100 CR + 760 AR ([96, 88, 107, 106, 103, 93, 96, 102, 102, 94, 88, 92, 103, 108, 109, 96, 106, 94, 93, 99, 105, 100, 94, 105, 102, 101, 101, 88, 88, 109]
)
    + result: 95% content relevant retrieved

**Comment by timm** [comment]
dont' understand 
"relevant retrieved (abs relevant as label)" vs
 "abs relevant retrieved (abs relevant as label)"

**Comment by azhe825** [comment]
Typos.

Just some working notes for additional discussion regarding to review comments.

What it says above is:
 - you can spend cost: 126 CR + 1370 AR to retrieve 98% recall (only 1 relevant missing)
 - or spend cost: 100 CR + 760 AR to retrieve 95% recall (4 relevant missing)

Where CR = cost for review content and AR = cost for review abstract
CR>>AR

**Comment by timm** [comment]
is CR "using all text in the paper" and AR "just using title+abstract"? if yes then then isn't your conclusion the reverse?

**Comment by azhe825** [comment]
No. CR is "the cost for a human to read all text in the paper and make classification". 
if a paper is reviewed and is "abstract relevant", its cost is AR+CR
if a paper is reviewed but is not "abstract relevant", its cost is AR

In either case, "just using title+abstract" for training.

**Comment by azhe825** [comment]
backup:

\parbox{.49\linewidth}{
\centering
\begin{subtable}
{\scriptsize \begin{tabular}{l@{~~~}l@{~~~}r@{~~~}r@{~~~}r@{~~~}r}
\arrayrulecolor{lightgray}
\multicolumn{2}{l}{\textbf{Kitchenham}}  & \multicolumn{2}{c}{\textbf{X95}} & \multicolumn{2}{c}{\textbf{WSS@95}}\\\hline
\textbf{Rank} & \textbf{Treatment} & \textbf{Median} & \textbf{IQR} & \textbf{Median} & \textbf{IQR} \\\hline
  1 &         HUSA &    760  &  190 & 0.50 & 0.16 \\
  1 &         HUTM &    810  &  120 & 0.47 & 0.08 \\
  1 &         HUSM &    800  &  210 & 0.47 & 0.12 \\
  1 &         PUTM &    840  &  200 & 0.45 & 0.12 \\
  1 &         HUTA &    850  &  100 & 0.45 & 0.06 \\
  1 &         PCTM &    860  &  130 & 0.44 & 0.09 \\
\hline  2 &         PUSA &    870  &  210 & 0.44 & 0.14 \\
  2 &         HCTA &    900  &  190 & 0.42 & 0.14 \\
  2 &         PUTA &    920  &  170 & 0.41 & 0.11 \\
  2 &         PUSM &    930  &  130 & 0.40 & 0.08 \\
  2 &         PCTA &    930  &  170 & 0.40 & 0.11 \\
  2 &         HCTM &    930  &  130 & 0.40 & 0.08 \\
\hline  3 &         HUSN &    1080  &  100 & 0.31 & 0.06 \\
\hline  4 &         PUSN &    1140  &  110 & 0.28 & 0.07 \\
  4 &         HUSW &    1150  &  30 & 0.27 & 0.02 \\
  4 &         HUTW &    1150  &  10 & 0.27 & 0.01 \\
  4 &         PUSW &    1160  &  170 & 0.26 & 0.10 \\
  4 &         HCTW &    1180  &  40 & 0.25 & 0.03 \\
  4 &         PCTW &    1190  &  170 & 0.25 & 0.10 \\
  4 &         PUTW &    1190  &  170 & 0.25 & 0.10 \\
\hline  5 &         HUTN &    1260  &  10 & 0.21 & 0.01 \\
  5 &         HCTN &    1280  &  30 & 0.20 & 0.02 \\
  5 &         PUTN &    1280  &  260 & 0.19 & 0.16 \\
  5 &         PCTN &    1290  &  260 & 0.19 & 0.15 \\
\hline  6 &         PCSW &    1370  &  260 & 0.14 & 0.22 \\
  6 &         PCSM &    1370  &  260 & 0.14 & 0.22 \\
  6 &         PCSN &    1400  &  340 & 0.12 & 0.22  \\
  6 &         PCSA &    1400  &  340 & 0.12 & 0.22  \\
\hline  7 &       linear &    1615  &  0 & 0 & 0 \\
\hline  8 &         HCSA &    1670  &  90 & -0.03 & 0.05 \\
  8 &         HCSN &    1670  &  90 & -0.03 & 0.05 \\
  8 &         HCSM &    1670  &  90 & -0.03 & 0.05 \\
  8 &         HCSW &    1670  &  90 & -0.03 & 0.05 \\
\hline \end{tabular}}
\end{subtable}}
